gradle/test.gradle

```diff
@@ -0,0 +1,55 @@
+dependencies {
+    // Jsonpath version is managed by the Spring Boot plugin
+    testImplementation group: 'com.jayway.jsonpath', name: 'json-path'
+    testImplementation group: 'io.opencensus', name: 'opencensus-impl', version: OPENCENSUS_VERSION
+    testImplementation group: 'org.apache.commons', name: 'commons-dbcp2', version: '2.7.0'
+    testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
+    // Jupiter version is managed by the Spring Boot plugin
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.+'
+    testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test'
+    testImplementation group: 'org.openapitools', name: 'jackson-databind-nullable', version: '0.2.1'
```

Comment:
I'm not totally sold on keeping the test dependencies in a separate place. I'm worried about the need to look at two files for dependencies. It's possible for dependencies to move from test-only to non-test overtime. It would probably be bad if we had a different `opencensus-impl` or `jackson-databind-nullable` in tests vs normally, but splitting it into two files feels like it's harder to catch that kind of thing.
Could be persuaded.

Reply:
SGTM, I was on the fence here, so your concern makes it an easy call to undo.